README.md

```diff
@@ -208,6 +208,26 @@ variant = determinator.which_variant(
 The `app_version` constraint for that flag needs to follow ruby gem version constraints. We support the following operators: `>, <, >=, <=, ~>`. For example:
 `app_version: ">=1.2.0"`
 
+### Using hooks for retriever
```

Comment:
Should we move this a bit further up, under the `Using over http` section?

Reply:
done